package.json

```diff
@@ -33,7 +33,6 @@
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-react": "^7.30.1",
     "eslint-plugin-react-hooks": "^4.6.0",
-    "git-secrets": "^0.0.11",
```

Comment:
This package has nothing to do with our `git-secrets` utility: https://github.com/awslabs/git-secrets

It was accidentally added, but should not be there